helper-cli/src/main/kotlin/commands/packageconfig/CreateCommand.kt

```diff
@@ -119,10 +119,10 @@ internal class CreateCommand : CliktCommand(
     private fun createPackageConfiguration(provenance: Provenance): PackageConfiguration =
         PackageConfiguration(
             id = packageId,
-            sourceArtifactUrl = provenance.let { it as? ArtifactProvenance }?.let {
+            sourceArtifactUrl = (provenance as? ArtifactProvenance)?.let {
                 it.sourceArtifact.url
             },
```

Comment:
This can be yet simpler as shown at https://github.com/oss-review-toolkit/ort/pull/5683#discussion_r954265714.